Gemfile.lock

```diff
@@ -535,5 +538,8 @@ DEPENDENCIES
   wirb
   wirble
 
+RUBY VERSION
+   ruby 2.4.2p198
```

Comment:
Is this safe?

Reply:
This is what bundler reported 😁 I think it's like the same kind of information as the `BUNDLED WITH` below: which tools I used to generate this.